clang/lib/Sema/SemaDecl.cpp

```diff
@@ -12796,6 +12799,7 @@ namespace {
     }
 
     void VisitCXXOperatorCallExpr(CXXOperatorCallExpr *E) {
+      llvm::SaveAndRestore CxxOpCallScope(isInCXXOperatorCall, true);
```

Comment:
We should be using [bugprone-argument-comment](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/argument-comment.html) style for all literal arguments like this.